### PR TITLE
Create startscreen

### PR DESCRIPTION
**Pull request by ingbeck**
Create a start sreen with a backgroundimage, logo, the appname and a login button

### CODE REVIEW

File: frontend/src/assets/startscreen_background.jpg
Comment (ReebalSami) [inline frontend/src/assets/startscreen_background.jpg:1]: sieht gut aus!! 👍

File: frontend/src/components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    children: ReactNode,
+}
+
+export default function Layout(props: Readonly<LayoutProps>) {
+    return (
+        <>
+            <main>
+                {props.children}
+            </main>
+        </>
```
Comment (ReebalSami) [inline frontend/src/components/Layout.tsx:13]: <>
</>

do you need these for the moment?

File: frontend/src/assets/startscreen_background.jpg
Comment (ingbeck) [reply frontend/src/assets/startscreen_background.jpg:1]: Kannst du nochmal reviewen? :) hab grad was umstellen müssen

File: frontend/src/pages/StartScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    return (
+        <>
+            <div className={"startpage"}>
+                <div className={"startpage-wrapper"}>
+                    <h1>Track My Meal</h1>
+                    <Avatar className={"startpage-wrapper-logo"}/>
+                    <Button variant="contained">Login</Button>
+                </div>
+            </div>
+        </>
```
Comment (ReebalSami) [inline frontend/src/pages/StartScreen.tsx]: here too.. :)

**Review by ReebalSami — APPROVED** [review]
otherwise LGTM 👍

**Pull request merged**